### ISSUE TITLE
Follow progress

### ISSUE THREAD

**Issue by darkdragon-001** [balthazar-space/balthazar#6]
I would like to follow progress / news (https://balthazar.space/wiki/News) via RSS-Feed in my feed reader. Could you please add this?

I tried to contact you via email about this matter without success

> SMTP error from remote server for RCPT TO command, host: mail.ljudmila.org (193.2.132.70) reason: 554 5.7.1 <info@balthazar.space>: Relay access denied

**Comment by FrostKnight** [comment]
I get the same issue when I try to email them...  weird right? 

Must be some weird glitch.

**Comment by FrostKnight** [comment]
Still looking at this, in case you guys get any closer, to your goal. :)

Btw, have you ever thought of utilizing, this as processor?

https://github.com/antonblanchard/microwatt

Dunno, how far its gotten, but I would love to have a mega ultra lightweight, power9 processor as an option for my next laptop.  ;) 

Just a thought, if you interested. 

Either way, I was curious if you guys still working on the balthazar device. :)

**Comment by NXT** [comment]
Hey, yes, sure we go on. just check new updated and third reiteration of our keyboard and a new yet unfinished HAT add-on.
Also thanks for the tip on Power9 - we rthink about and also evaluate ULX4M-CM4-HAT designed by Radiona.org which is very interesting too.

**Comment by FrostKnight** [comment]
> Hey, yes, sure we go on. just check new updated and third reiteration of our keyboard and a new yet unfinished HAT add-on. Also thanks for the tip on Power9 - we rthink about and also evaluate ULX4M-CM4-HAT designed by Radiona.org which is very interesting too.

Microwatt does look interesting, but you do need to check with their developers, to see if it is at the level you desire, in being lightweight, secure, fast, potential in general...

Aka, you want it to be better than arm64 even in its newer generations when it comes to all 3, if possible.

Though if it is even at the speed of gen 3 intel, like that of a thinkpad x230, or faster, I would be mega interested. But yeah, I would want it to at least be mega ultra lightweight. 

:)

Please though, let me know if it is usable without blobs or backdoors and if it is more stable and lightweight in watts needed and if its at least 2x-4x faster than LS1028A.

btw, I had asked this before,  but it seems, mnt reform devs are going to attempt this in the future:

they plan to make a 7 inch device, no idea how much it would cost though. ;)

I wondered, if this interests you at all, because perhaps, you could do the same in the future.

Although if not, I hope you will still ask about the Power9 microwatt processor's level of completion.

I am very curious aka...

I tried to ask once, but no one has answered yet. Maybe someone who is potential buyer, could get some info.

:)

Hope you guys do well regardless! :)

**Comment by NXT** [comment]
Thanks for your info and thoughts. We aim at about 300 Euros device that is a cross between old OLPC concept and ThinkPad in exactly x2xx range. Issues we are facing is availability of ICs and overall lack of abilities to secure working CPU/SoM that would fit to our purpose. RiscV people are rather far behind promised useful CPUs and all other are just prototypes and not to avail.
We try really hard indeed. Maybe this FPGA might be the way if as nothing as a coprocessor that can make balthazar more appealing. Really thanks a lot for your suggestions.

**Comment by FrostKnight** [comment]
Btw, not sure if this project, is the one you want, but its possible, you could do your own implementation of it:

https://libre-soc.org/

Or, you could scratch each other's backs so to speak...

:)

Or do your own version, etc...

Just some thoughts.

**Comment by FrostKnight** [comment]
> Thanks for your info and thoughts. We aim at about 300 Euros device that is a cross between old OLPC concept and ThinkPad in exactly x2xx range. Issues we are facing is availability of ICs and overall lack of abilities to secure working CPU/SoM that would fit to our purpose. RiscV people are rather far behind promised useful CPUs and all other are just prototypes and not to avail. We try really hard indeed. Maybe this FPGA might be the way if as nothing as a coprocessor that can make Balthazar more appealing. Really thanks a lot for your suggestions.

I wonder how microwatt though is doing compared to risc-v. They basically both have similar goals, microwatt is also very low on battery usage just like risc-v and I could go on and on. I am not sure how ready they are either, but it definitely has my interest. An improvement to say the least over other options with the exception of the one I mentioned above this specific comment.

If you plan to make a quad core processor with the speed of dual core P8600 from intel, aka, similar abilities, but better usage, even that would be good enough for people like me. But I am sure that even won't be an easy road. Trackpoint would also be very interesting as well as a feature. Anywho, that's all I have for now as thoughts.

**Comment by NXT** [comment]
> I would like to follow progress / news (https://balthazar.space/wiki/News) via RSS-Feed in my feed reader. Could you please add this?
> 
> I tried to contact you via email about this matter without success
> 
> > SMTP error from remote server for RCPT TO command, host: mail.ljudmila.org (193.2.132.70) reason: 554 5.7.1 [info@balthazar.space](mailto:info@balthazar.space): Relay access denied

Yes, thanks for noticing and a sorry about that It should be rectified by now. It was the bad DNS.
Also, the new website is coming up soon, so RSS will be included.
We were advised not to touch the MediaWiki installation. 
In case you ask about the development, it still goes on, at about somewhat slower pace - but the main reason iis, 
mostly waiting for the parts at the small volume, a warning not to use HDMI from a lawyer. 
we will do it surely till the end, but as we claimed - we do not produce Balthazar, we are the design proposal for the EU initiative to have some electronic manufacturing back to EU. There will be small start-ups producing our device, but hard to tell at which price. We always aimed at 300 Euros.
